### ISSUE TITLE
How to make data augmentation

### ISSUE THREAD

**Issue by E-MHJ** [iro-cp/FCRN-DepthPrediction#37]
Hi ,I am trying to recreate your results on the NYU_V2 dataset，but the huber loss can't converge when training. I guess i need to make data augmentation, but i am unsure how to do it.  Now i have 12k image pairs, if i want to get 95K pairs, should i make rotation, scaling, color transformations and flip eight times 
for each RGB-D image pair ? Besides, when computing loss, if the input groundtruth depth size is 640x480? 
Thank you.

**Comment by Ariel-JUAN** [comment]
@E-MHJ  you need data augmentation, following as Elign's paper. Besides, can you show me you loss code? My loss converge but the result is not very good.

**Comment by mortalsoulzy** [comment]
@Ariel-JUAN Hi,I'm trying to reproduce this experiment with tensorflow , but after training for few steps ,the output depth map convergence to the same number for all pixels.And don't change anymore.I'm quite confused for that,can u share some training details with me?That would be of great help,Tanks~

**Comment by Ariel-JUAN** [comment]
I don't really understand your question...... maybe you can leave you email and I will contact you.

**Comment by mortalsoulzy** [comment]
@Ariel-JUAN My email address is mortalsoulzy@gmail.com，thank u so much~
